test/preview_test.dart

```diff
@@ -0,0 +1,50 @@
+import 'dart:io';
+
+import 'package:nextcloud/nextcloud.dart';
+import 'package:test/test.dart';
+
+import 'config.dart';
+
+void main() {
+  final config = getConfig();
+  final client = getClient(config);
+
+  group('Preview', () {
+    final fullImagePath = '${config.testDir}/preview.png';
+    final rootWithoutUser =
+        config.testDir.split('/files/${config.username}')[1];
+    final imageRootPath = '$rootWithoutUser/preview.png';
+    WebDavFile previewFile;
+
+    setUpAll(() async {
+      await client.webDav
+          .upload(File('test/files/test.png').readAsBytesSync(), fullImagePath);
```

Comment:
Test file is not found from root.

Reply:
No, sry, I think it has actually to do with the parallel execution when running all tests.